full-stack-tests/rpc-interface/src/frontend/setup/TestContext.ts

```diff
@@ -60,16 +58,17 @@ export class TestContext {
     Logger.initializeToConsole();
     Logger.setLevelDefault(this.settings.logLevel === undefined ? LogLevel.Warning : this.settings.logLevel);
 
-    if (undefined !== this.settings.oidcClientId) {
-      this.adminUserAccessToken = await getAccessTokenFromBackend({
-        email: this.settings.users[0].email,
-        password: this.settings.users[0].password,
-      } as TestUserCredentials, {
-        clientId: this.settings.oidcClientId,
-        redirectUri: this.settings.oidcRedirect,
-        scope: this.settings.oidcScopes,
-        authority: this.settings.oidcAuthority,
-      } as TestBrowserAuthorizationClientConfiguration);
+    if (undefined !== this.settings.clientConfiguration!.clientId) {
+      this.serviceAuthToken = await getServiceAuthTokenFromBackend({
+        clientId: this.settings.clientConfiguration!.clientId,
+        clientSecret: this.settings.clientConfiguration!.clientSecret,
```

Comment:
you should check if this is defined as well on L61

Reply:
> you should check if this is defined as well on L61

added the check inside the if statement